server/src/main/java/p2/service/impl/InterestService.java

```diff
@@ -26,20 +26,31 @@ public static Interest findById(int id) {
 		return interestDAO.findById(id);
 	}
 
-	public static void deleteById(int id) {
-		interestDAO.deleteById(id);
+	public static boolean deleteById(int id) {
+		return interestDAO.deleteById(id);
 	}
 
 	public static List<Interest> findByProductId(int productId) {
 		return interestDAO.findByProductId(productId);
 	}
 
-	public static int getNumberOfInterest_ByProductId(int productId) {
+	public static int getNumberOfInterestByProductId(int productId) {
 		int count = 0;
 		for (Interest interest : findByProductId(productId))
 			count += interest.getQuantity();
 		return count;
 
 	}
 
+	public static int changeStatusNotEnoughInterest(int productId) {
+		return interestDAO.changeStatus(productId, InterestStatus.NOT_ENOUGH_INTEREST.value);
+	}
+
+	public static int changeStatusCancelledBySeller(int productId) {
+		return interestDAO.changeStatus(productId, InterestStatus.CANCELLED_BY_SELLER.value);
+	}
+
+	public static int changeStatusDealComplted(int productId) {
```

Comment:
change this to changeStatusToDealCompleted

Reply:
Done